.github/dockerfiles/Dockerfile.clang

```diff
@@ -1,4 +1,4 @@
-FROM debian:buster AS build
+FROM debian:12 AS build
```

Comment:
I think this is the old docker images, why we don't drop this?

Reply:
No, this is used as part of the checks workflow. It actually _should_ have been getting updated properly on every Debian release, but that’s obviously not happened.